torch/_inductor/kernel/flex_attention.py

```diff
@@ -387,10 +387,10 @@ def build_subgraph_buffer(
     (torch.float32, 128): (128, 32, 4, 3),
     (torch.float32, 256): (64, 16, 4, 3),
     (torch.bfloat16, 64): (128, 64, 4, 3),
-    (torch.bfloat16, 128): (128, 128, 8, 2),
+    (torch.bfloat16, 128): (128, 64, 8, 3),
```

Comment:
was this just more tuning?

Reply:
Yeah. Also, to avoid the segfaults I'm avoiding `num_stages=2`.